src/WebApi/Domain/Models/InterviewQuestionModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Domain.Models
+{
+    public class InterviewQuestionModel
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
```

Comment:
Please fix formatting and add newlines between props so its consistent between Model and Entity class 😊